crates/control-interface/src/types.rs

```diff
@@ -433,29 +433,53 @@ pub struct LinkDefinition {
     pub values: LinkSettings,
 }
 
+/// Identifier of one or more entities on the lattice used for addressing. May take many forms, such as:
+/// - actor public key
+/// - provider public key
+/// - opaque string
+pub type LatticeTargetId = String;
```

Comment:
I want to noodle on this name a bit, it's a little strange to provide a target ID to the `source_id` struct member. I've been calling these component IDs, but maybe that needs to be more clear.

We can address this w/ conversation and a follow up, since it's just String aliasing 😄

Reply:
Yeah I agree -- honestly we should have a proper `enum` here at some point since we can somewhat identify *some* of them -- but yeah maybe this should be like `LatticeRouteName`.  Probably `Name` and not `Id` because we might end up giving every route an individual ID (like ULID/UUID)..

I'm gonna change it to that actually

- [ ] `LatticeTargetId` -> `LatticeRouteName`

Reply:
Just chatted about this, landed on `ComponentId` for source, and `LatticeTarget` for target